app/components/Layout/Header.tsx

```diff
@@ -33,7 +33,7 @@ const HeaderLayout = ({isLoggedIn = false, isRegistered = false, children}) => {
   return (
     <header>
       <div className="nav-header">
-        <nav>
+        <nav aria-label="Main">
```

Comment:
Rule: If multiple `<nav>` (aka. a page / accessibility landmark) elements exist, they must have an `aria` attribute set which labels them as distinct. 

When navigating via a screenreader, it's possible to go direct to a `<nav>` and this will let it announce the two `<nav>`s (one here, one in Subheader) by their distinct name. Announced as: "Main navigation", "Secondary navigation" as labelled.